pkg/api/admin_users_test.go

```diff
@@ -28,6 +28,20 @@ const (
 	existingTestLogin = "existing@example.com"
 )
 
+type mockAuthInfoService struct {
```

Comment:
I use also this service, we can put a service mock afterward when the entire bus is gone, no rush